### ISSUE TITLE
Problems after last update

### ISSUE THREAD

**Issue by tuch96** [Donnnno/Arcticons#100]
After the Update to 1.4.6 my calender icon shows only the original app icon and not the arcticons version. Please can you fix this problem? Thanks!!

<item component="ComponentInfo{com.simplemobiletools.calendar.pro/com.simplemobiletools.calendar.pro.activities.SplashActivity.Orange}" drawable="kalender"/>

**Comment by Donnnno** [comment]
Going to take a look at this. Probably has something to do with the new dynamic calendar icons.

**Comment by Donnnno** [comment]
Hi, I don't see anything wrong in the code right now. What launcher are you using?

**Comment by tuch96** [comment]
Open Launcher. The problem is also present with other calender apps.

**Comment by Donnnno** [comment]
I think I've found the problem for the normal calendar, not sure about simple calendar tho.

**Comment by Donnnno** [comment]
Let me know if it's fixed in the 1.5.0 release!

**Comment by tuch96** [comment]
I will inform you as soon as I have tested the f-droid version!! I don't have an update notification yet. Thanks for your brilliant support!!!

**Comment by tuch96** [comment]
Unfortunately, it does not work with the new version either. I have now switched back to version 1.3.5.

**Comment by Donnnno** [comment]
Hmm strange, I've tested a bunch of icon packs to see where the problem lies. Right now every launcher seems to work fine, some use the dynamic calendar icons and others the normal fallback, except for Open Launcher. I dunno why tho. 

Maybe the best thing to do is open an issue over at open launcher's repo because everything should work from my end: https://github.com/OpenLauncherTeam/openlauncher

**Comment by tuch96** [comment]
I open an issue on the Open Launcher Github site. Cross the fingers for a solution :-).

**Comment by Donnnno** [comment]
Yeah, I noticed it. Open Launcher isn't actively developed anymore, so I don't think there will be any solution anytime soon.

Switch to a different launcher would be the easiest solution.
